aten/src/ATen/native/nested/NestedTensorMath.cpp

```diff
@@ -945,6 +947,11 @@ Tensor _NestedTensor_GeneralizedBMM(const Tensor& self, const Tensor& mat2) {
   return output;
 }
 
+Tensor& matmul_out_nested(const Tensor& tensor1, const Tensor& tensor2, Tensor& result) {
```

Comment:
this is NOT how out function works!
You MUST write into the result that was given to you.

You can check other out variants but you most likely need to resize and then copy_ into result here.

Reply:
I see. That's the very old-school c semantics `void matmul_out(const double* tensor1, const double* tensor2, double* result)` then

Reply:
Similar yes